### ISSUE TITLE
Document how to setup git (username + email)

### ISSUE THREAD

**Issue by scottgonzalez** [jquery/contribute.jquery.org#40]
https://help.github.com/articles/set-up-git#set-up-git

**Comment by scottgonzalez** [comment]
We should link to this from Contributing to Code, Commits and Pull Requests, and even the CLA page.

**Comment by arthurvr** [comment]
Honestly, should our contribution guidelines go that far? Might as well link up other good resources (such as the GitHub docs). I just feel like this is enough documented on other places. IMHO [the CLA page](https://github.com/jquery/contribute.jquery.org/blob/master/pages/CLA/status.md#updating-git-config) is enough for us.

**Comment by scottgonzalez** [comment]
> Honestly, should our contribution guidelines go that far? Might as well link up other good resources (such as the GitHub docs)

That's exactly what this issue is, linking to GitHub from the relevant pages on our site.

**Comment by arthurvr** [comment]
> That's exactly what this issue is, linking to GitHub from the relevant pages on our site.

Oh, misunderstood you, I though you wanted the content to literally be documented on the site.

**Comment by scottgonzalez** [comment]
Nope. I'm definitely on the same page as you, we should be linking to existing content when it's available.
